dev/org.eclipse.codewind.core/src/org/eclipse/codewind/core/internal/InstallUtil.java

```diff
@@ -260,16 +278,25 @@ public static Process runInstaller(String cmd, String... options) throws IOExcep
 		return builder.start();
 	}
 	
-	public static String getInstallerExecutable() throws IOException {
-		if (installExec != null && (new File(installExec)).exists()) {
-			return installExec;
+	public static String getInstallerExecutable(InstallOperation operation) throws IOException {
+		String installPath = operation.getInstallPath();
+		if (installPath != null && (new File(installPath).exists())) {
+			return installPath;
 		}
-		
+
 		// Get the current platform and choose the correct executable path
 		OperatingSystem os = PlatformUtil.getOS(System.getProperty("os.name"));
-		String relPath = installMap.get(os);
+		
+		Map<OperatingSystem, String> osPathMap = operation.getOSPathMap();
+		if (osPathMap == null) {
```

Comment:
This would never be `null` right?  Since we control how the operations are instantiated, it's all private.